src/accessors/getValidations.ts

```diff
@@ -4,7 +4,7 @@ import type { SchemaNodeKind } from '../nodes/types';
 import type { SchemaFragment } from '../types';
 import { pick } from '../utils/pick';
 
-export const COMMON_VALIDATION_TYPES: string[] = ['readOnly', 'writeOnly'];
+export const COMMON_VALIDATION_TYPES: string[] = ['readOnly', 'writeOnly', 'style'];
```

Comment:
I added this little guy - hope it's OK. This allows us to completely ditch the custom parameter rendering we have in elements (only thing missing was tracking the `style` option that http parameters sometimes use).

Reply:
yeah, fair enough.